content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -266,6 +266,7 @@ different Kubernetes components.
 | `EvenPodsSpread` | `true` | Beta | 1.18 | 1.18 |
 | `EvenPodsSpread` | `true` | GA | 1.19 | - |
 | `ExecProbeTimeout` | `true` | GA | 1.20 | - |
+| `ExternalPolicyForExternalIP` | `true` | GA | 1.18 | 1.22 |
```

Comment:
This should be:
```suggestion
| `ExternalPolicyForExternalIP` | `true` | GA | 1.18 | - |
```
Was this feature gate beta for graduating to stable? If so, there should also be a previous row for the beta phase. Same story if the feature gate started life as alpha.